contrib/python-syrupy/template.py

```diff
@@ -0,0 +1,21 @@
+pkgname = "python-syrupy"
+pkgver = "4.6.1"
+pkgrel = 0
+build_style = "python_pep517"
+hostmakedepends = [
+    "python",
+    "python-build",
+    "python-installer",
+    "python-poetry-core",
+    "python-setuptools",
+    "python-wheel",
+]
+checkdepends = ["python-pytest", "python-syrupy"]
+pkgdesc = "Pytest snapshot plugin"
+maintainer = "ttyyls <contact@behri.org>"
+license = "Apache-2.0"
+url = "https://tophat.github.io/syrupy"
+source = f"$(PYPI_SITE)/s/syrupy/syrupy-{pkgver}.tar.gz"
+sha256 = "37a835c9ce7857eeef86d62145885e10b3cb9615bc6abeb4ce404b3f18e1bb36"
+# broken tests
+options = ["!check"]
```

Comment:
in what way? usually fine to skip some

Reply:
iirc missing another 2 layers of dependencies and some were broken in 3.11, i could probably just skip the one test that depends on it in tldextract and skip the check for it since it's only relevant for the upstream release process... or chase after all the revdeps and patch them where necessary...

Reply:
ah hm sounds like a lot :)

Reply:
i could just change it to "unpackaged dependencies" and sort it out later

Reply:
yeah seem better